HintList.py

```diff
@@ -198,79 +221,763 @@ def getRequiredHints(world):
+
+    'Dodongos Cavern Boss Room Chest':                             ("#above King Dodongo# lies", None, 'exclude'),
+
+    'Dodongos Cavern Map Chest':                                   ("a #muddy wall in Dodongo's Cavern# hides", None, 'exclude'),
+    'Dodongos Cavern Compass Chest':                               ("a #statue in Dodongo's Cavern# guards", None, 'exclude'),
+    'Dodongos Cavern Bomb Flower Platform Chest':                  ("above a #maze of stone# in Dodongo's Cavern lies", None, 'exclude'),
+    'Dodongos Cavern Bomb Bag Chest':                              ("the #second lizard cavern battle# yields", None, 'exclude'),
+    'Dodongos Cavern End of Bridge Chest':                         ("a #chest at the end of a bridge# yields", None, 'exclude'),
+
+    'Dodongos Cavern MQ Map Chest':                                ("a #muddy wall in Dodongo's Cavern# hides", None, 'exclude'),
+    'Dodongos Cavern MQ Bomb Bag Chest':                           ("an #elevated alcove# in Dodongo's Cavern holds", None, 'exclude'),
+    'Dodongos Cavern MQ Compass Chest':                            ("#fire-breathing lizards# in Dodongo's Cavern guard", None, 'exclude'),
+    'Dodongos Cavern MQ Larvae Room Chest':                        ("#baby spiders# in Dodongo's Cavern guard", None, 'exclude'),
+    'Dodongos Cavern MQ Torch Puzzle Room Chest':                  ("above a #maze of stone# in Dodongo's Cavern lies", None, 'exclude'),
+    'Dodongos Cavern MQ Under Grave Chest':                        ("#beneath a headstone# in Dodongo's Cavern lies", None, 'exclude'),
+
+    'Jabu Jabus Belly Map Chest':                                  ("#tentacle trouble# in a deity's belly guards", "a #slimy thing# guards", 'exclude'),
+    'Jabu Jabus Belly Compass Chest':                              ("#bubble trouble# in a deity's belly guards", "#bubbles# guard", 'exclude'),
+
+    'Jabu Jabus Belly MQ First Room Side Chest':                   ("shooting a #mouth cow# reveals", None, 'exclude'),
+    'Jabu Jabus Belly MQ Map Chest':                               (["#pop rocks# hide", "an #explosive palate# holds"], "a #boulder before cows# hides", 'exclude'),
+    'Jabu Jabus Belly MQ Second Room Lower Chest':                 ("near a #spiked elevator# lies", None, 'exclude'),
+    'Jabu Jabus Belly MQ Compass Chest':                           ("a #drowning cow# unveils", None, 'exclude'),
+    'Jabu Jabus Belly MQ Second Room Upper Chest':                 ("#moving anatomy# creates a path to", None, 'exclude'),
+    'Jabu Jabus Belly MQ Basement Near Switches Chest':            ("a #pair of digested cows# hold", None, 'exclude'),
+    'Jabu Jabus Belly MQ Basement Near Vines Chest':               ("a #pair of digested cows# hold", None, 'exclude'),
+    'Jabu Jabus Belly MQ Near Boss Chest':                         ("the #final cows' reward# is", None, 'exclude'),
```

Comment:
I think this hint ought to be clearer it's about Jabu Jabu (and not say, 5 cows on ZR wanting to hear songs).

Reply:
Done